profiles/artimanputra.json

```diff
@@ -1,6 +1,6 @@
 {
-    "image": "https://user-images.githubusercontent.com/79042027/145818832-0bdad31d-7dfb-423f-8c76-f51a14c202e1.png",
+    "image": "https://user-images.githubusercontent.com/91655303/145820969-c180ba53-ed0d-4992-8bc2-f40c53b05f3b.png",
     "issueId": 901,
     "name": "Arti Manputra",
     "username": "artimanputra"
-  }
\ No newline at end of file
+}
```

Comment:
```suggestion
}

```

Reply:
> 

Can you please tell me what I have to do like I am not getting anything?

Reply:
new line